api/v1/verticadb_webhook.go

```diff
@@ -818,6 +821,26 @@ func (v *VerticaDB) hasValidClientServerTLSMode(allErrs field.ErrorList) field.E
 	return allErrs
 }
 
+// hasTLSSecretsSetForRevive checks whether the TLS secrets are set for the revive init policy
+// when TLS is enabled
+func (v *VerticaDB) hasTLSSecretsSetForRevive(allErrs field.ErrorList) field.ErrorList {
+	if vmeta.UseTLSAuth(v.Annotations) && v.Spec.InitPolicy == CommunalInitPolicyRevive {
+		if v.GetHTTPSNMATLSSecret() == "" {
```

Comment:
```suggestion
		if v.GetHTTPSNMATLSSecret() == "" && v.Spec.NMATLSSecret == "" {
```

We still support v.Spec.NMATLSSecret == ""

Reply:
Will we copy v.Spec.NMATLSSecret to v.Spec.HTTPSNMATLS.Secret? Should we use "or" condition?